Misc/NEWS.d/next/Core and Builtins/2020-10-31-17-50-23.bpo-42218.Dp_Z3v.rst

```diff
@@ -0,0 +1,2 @@
+Check for errors in left-recursive parser rules to avoid cases where such errors do not
+get handled in time and appear as long-distance bugs in other places.
```

Comment:
```suggestion
Fixed a bug in the PEG parser that was causing crashes in debug mode. Now errors are checked
in left-recursive rules to avoid cases where such errors do not get handled in time and appear
as long-distance crashes in other places.
```

Reply:
Feel free to ignore the suggestion, it is mainly to make the message slightly more clear to readers not familiar with the parser internals.

Reply:
Looks better. Thanks!